resources/eventing/charts/nats/crds/customeresourcedefinition.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Let's also put the CRDs behind the feature flag ( I mean `install.enabled`).

Reply:
Right, done

Reply:
See: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/
Templating is not supported in CRDs.